dev/com.ibm.ws.jaxws.ejb_fat/publish/servers/com.ibm.ws.jaxws.ejb.fat.ejbwsprovider/bootstrap.properties

```diff
@@ -1 +1,2 @@
-bootstrap.include=../testports.properties
\ No newline at end of file
+bootstrap.include=../testports.properties
+com.ibm.ws.logging.trace.specification=org.apache.cxf.*=all=enabled:com.ibm.ws.jaxws.*=all=enabled
```

Comment:
Is it necessary to have this trace enabled all of the time and increase the output?  Could it maybe be enabled only when debugging?

Reply:
Good point, it's not necessary, I will remove this with another PR

Reply:
Already removed this trace